src/XMLimport.cpp

```diff
@@ -172,12 +172,18 @@ bool XMLimport::importPackage(QFile* pfile, QString packName, int moduleFlag, QS
                     /*||(mVersionMajor==1&&mVersionMinor)*/) {
                     // Minor check is not currently relevant, just abort on 2.000f or more
 
-                    QString moanMsg = tr("[ ALERT ] - Sorry, the file being read:\n"
-                                         "\"%1\"\n"
-                                         "reports it has a version (%2) it must have come from a later Mudlet version,\n"
-                                         "and this one cannot read it, you need a newer Mudlet!")
-                                              .arg(pfile->fileName(), versionString);
-                    mpHost->postMessage(moanMsg);
+                    if (pErrorMessage) {
+                        *pErrorMessage = QStringLiteral("failed to import package as it has a version number of %1 which is too late to "
```

Comment:
What does a 'late version number' mean? I think the earlier 'too new' makes more sense.

Reply:
"failed to import package as it has a version number of %1 which is too late to "
    "be understood by this version of Mudlet, it seems you need a newer one"

How about:

    "failed to import package as it has a version number of %1 which is too new to "
    "be understood by this version of Mudlet, it seems you need a more recent "
    "edition of this MUD client"

I didn't want to reuse the word Mudlet again, and saying "you need a newer one" could be read
ambiguously to mean either _Mudlet_ or _the package_ when we mean the former not the latter.

Of course - the fact that a more recent edition must exist to produce the package means that a friendly suggestion to update is not necessarily a bad thing... :smirk: